### PR TITLE
feat:done with list and text-utils

### PR DESCRIPTION
**Pull request by tushargoyal812**
added list and text utilities

https://user-images.githubusercontent.com/69681896/153345451-e286cc73-a380-4aed-936a-5f46ab46a090.mp4

https://user-images.githubusercontent.com/69681896/153345518-9bd94e86-1b97-4b2a-b837-5b703565d3ea.mp4

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+.list-child {
+    padding: 0.5rem;
+    cursor: pointer;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+}
+
+.list-child:hover {
+    background-color: rgb(233, 225, 225);
```
Comment (bishnoimukesh) [inline components/list/list.css]: use var for color for consistency as you used above

File: components/list/list.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                src="https://carbon.now.sh/embed?bg=rgba%2855%2C52%2C18%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522simple-list%2522%253E%250A%2520%2520%253Ch2%2520class%253D%2522list-heading%2522%253Esimple%2520list%253C%252Fh2%253E%250A%2520%2520%253Chr%253E%250A%2520%2520%253Cul%2520class%253D%2522list-parent%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-child%2522%253Elist%2520item%25201%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-child%2522%253Elist%2520item%25202%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-child%2522%253Elist%2520item%25203%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-child%2522%253Elist%2520item%25204%253C%252Fli%253E%250A%2520%2520%253C%252Ful%253E%250A%253C%252Fdiv%253E"
+                style="width: 100%; height: 245px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
```
Comment (bishnoimukesh) [inline components/list/list.html:96]: avoid inline styling

File: components/list/list.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                style="width: 100%; height: 394px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
```
Comment (bishnoimukesh) [inline components/list/list.html:125]: same for this

File: components/text-utils/text-utils.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                <div class="m-0-25"></div>
+                <h5>This is heading 5</h5>
+                <div class="m-0-25"></div>
+                <h6>This is heading 6</h6>
+            </div>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%2855%2C52%2C18%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253EThis%2520is%2520heading%25201%253C%252Fh1%253E%250A%253Ch2%253EThis%2520is%2520heading%25202%253C%252Fh2%253E%250A%253Ch3%253EThis%2520is%2520heading%25203%253C%252Fh3%253E%250A%253Ch4%253EThis%2520is%2520heading%25204%253C%252Fh4%253E%250A%253Ch5%253EThis%2520is%2520heading%25205%253C%252Fh5%253E%250A%253Ch6%253EThis%2520is%2520heading%25206%253C%252Fh6%253E"
+                style="width: 100%; height: 172px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
```
Comment (bishnoimukesh) [inline components/text-utils/text-utils.html:101]: same for this

File: components/text-utils/text-utils.html [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+                we can use small text when we want.
+            </p>
+            <div class="heading-container mg-auto flex flex-direction-col align-items justify-content p-2">
+                <small>This is small</small>
+            </div>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%2855%2C52%2C18%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Csmall%253EThis%2520is%2520small%253C%252Fsmall%253E"
+                style="width: 100%; height: 79px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
```
Comment (bishnoimukesh) [inline components/text-utils/text-utils.html:114]: same

File: components/text-utils/text-utils.html [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+                You can use 'center-text' class and make your texts center in your page.
+            </p>
+            <div class="heading-container mg-auto p-2">
+                <div class="center-text">this is a center text</div>
+            </div>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%2855%2C52%2C18%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522center-text%2522%253Ethis%2520is%2520a%2520center%2520text%253C%252Fdiv%253E"
+                style="width: 100%; height: 79px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
```
Comment (bishnoimukesh) [inline components/text-utils/text-utils.html:140]: same

File: variable.css [hunk 7]
```diff
@@ -7,4 +7,5 @@
     --danger-color:#F44336;
     --warning-color:#FF9800;
     --information-color:lightseagreen;
+    --gray-color:gray;
```
Comment (bishnoimukesh) [inline variable.css:10]: either use hex code or name while defining

**Review by bishnoimukesh — COMMENTED** [review]
take care of the suggested changes. rest everything seems fine

File: components/list/list.css (hunk 1 above)
Comment (tushargoyal812) [reply components/list/list.css]: done

File: components/list/list.html (hunk 2 above)
Comment (tushargoyal812) [reply components/list/list.html:96]: it was by default from carbon

File: components/list/list.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
+@import url('/index.css');
+
+
+.list-container {
+    width: 28rem;
+    padding: 2rem;
+    border: 2px solid var(--gray-color);
+    border-radius: 5px;
+    background-color: var(--bg-color);
+}
```
Comment (yogesh-) [inline components/list/list.css:10]: use rem everywhere.

File: util.css [hunk 9]
```diff
@@ -55,6 +55,10 @@ iframe {
 
 /* margin utils */
 
+.m-0-25{
+    margin: 0.25rem;
+}
+
 .m-1 {
```
Comment (yogesh-) [inline util.css:62]: dont use numbers in class names.

**Review by yogesh- — APPROVED** [review]
looks good to merge

File: components/list/list.css (hunk 8 above)
Comment (tushargoyal812) [reply components/list/list.css:10]: ok

File: util.css (hunk 9 above)
Comment (tushargoyal812) [reply util.css:62]: ok

**Pull request merged**